### PR TITLE
Add Sylius resource bundle

### PR DESCRIPTION
**Pull request by loic425**
| Q             | A
| ------------- | ---
| License       | MIT

### CODE REVIEW

File: sylius/resource-bundle/1.6/config/packages/sylius_resource.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+sylius_resource:
+    # Override default settings
+    settings:
+        #paginate: null
+        #limit: null
+        #allow_paginate: [10, 20, 30]
+        #default_page_size: 10
+        #sortable: null
+        #filterable: false
+        #criteria: null
```
Comment (Nyholm) [inline sylius/resource-bundle/1.6/config/packages/sylius_resource.yaml:11]: There is no need to show all default config here. That is what your docs is for. 

Please remove.

**Review by Nyholm — CHANGES_REQUESTED** [review]
Just one small thing, Im happy after that =)

**Comment by Nyholm** [comment]
Small ping

File: sylius/resource-bundle/1.6/config/packages/sylius_resource.yaml (hunk 1 above)
Comment (loic425) [reply sylius/resource-bundle/1.6/config/packages/sylius_resource.yaml:11]: @Nyholm I just keep "settings" key to show it cause I think a lot of Sylius user doesn't know you can override them.

File: mobizel/markdown-docs-bundle/0.1/manifest.json [hunk 2]
```diff
@@ -1,4 +1,7 @@
 {
+    "container": {
+        "locale": "en"
```
Comment (Nyholm) [inline mobizel/markdown-docs-bundle/0.1/manifest.json:3]: Lets keep the changes in this PR to Sylius resource bundle.

**Comment by loic425** [comment]
@Nyholm I think this is ok.
I just tested with the following commands:
```bash
symfony new test-resource-bundle --full
cd test-resource-bundle
composer config extra.symfony.allow-contrib true
export SYMFONY_ENDPOINT=https://flex.symfony.com/r/github.com/symfony/recipes-contrib/1039
composer req "sylius/resource-bundle:^1.6"
```

And cache clear command is ok :)

**Review by Nyholm — APPROVED** [review]
Im happy with this. 

You may need to rebase the PR so the new validation kick in.

**Comment by Nyholm** [comment]
No rebase needed. Thank you

**Pull request merged**